src/metrics/remote_peer_info.go

```diff
@@ -0,0 +1,84 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/protolambda/zrnt/eth2/beacon"
+)
+
+// BASIC HOST INFO
+
+// BasicHostInfo contains the basic Host info that will be requested from the identification of a libp2p peer
+type BasicHostInfo struct {
+	TimeStamp time.Time
+	// Peer Host/Node Info
+	PeerID          string
+	NodeID          string
+	UserAgent       string
+	ProtocolVersion string
+	Addrs           string
+	PubKey          string
+	RTT             time.Duration
+	Protocols       []string
+	// Information regarding the metadata exchange
+	Direction string
+	// Metadata requested
+	MetadataRequest bool
+	MetadataSucceed bool
+}
+
+// BEACON METADATA
+
+// Basic BeaconMetadata struct that includes the timestamp of the received beacon metadata
+type BeaconMetadataStamped struct {
```

Comment:
shouldnt this be part of the peer.go?

Reply:
I don't have any strong opinion about having it on the same peer.go file, I thought that it could look cleaner separating it. If you think that it belongs to the peer.go I can move it there.